packages/create-mc-app/src/process-options.ts

```diff
@@ -59,6 +60,26 @@ const getInitialProjectKey = async (
   return initialProjectKey;
 };
 
+const getCloudIdentifier = async (
+  rl: Interface,
+  options: TCliCommandOptions
+) => {
+  if (options.cloudIdentifier) {
+    throwIfCloudRegionNotSupported(options.cloudIdentifier);
+    return options.cloudIdentifier;
+  }
+
```

Comment:
Nit: we can also return the default in case the option `yes` was provided

```suggestion

if (options.yes) {
  return supportedCloudRegions['gcp-eu'];
}

```

Reply:
Updated in this commit: [1dc901c](https://github.com/commercetools/merchant-center-application-kit/pull/3564/commits/1dc901c3d497af7cc5c144f61f234ae3a4e1f0c3)